tests/ignite/distributed/utils/__init__.py

```diff
@@ -228,6 +228,31 @@ def _test_distrib_barrier(device):
     assert tt.item() == true_res + 10.0
 
 
+def _test_distrib_new_group(device):
```

Comment:
Can you please add a test to check `ValueError("Group should be list of int")`. Thanks